### DISCUSSION TITLE
iOS App > Share via > Email

### DISCUSSION THREAD

**Discussion by MFKDGAF** [community/community#139726] [Mobile]
### Select Topic Area

Product Feedback

### Body

When sharing a repository via email, the default action is not to add a subject to the email but only to add a link to the repository in the body of the email.

I would like to propose that the default action should be to fill in the subject of the email with the name of the repository or the author + name of the repository.

An example of this would be on the iOS Reddit mobile app when sharing a post (not a comment), the default action is to add the name of the post to the subject of the email.

**Comment by HenSquared** [comment]
Good feedback @MFKDGAF. Thank you for the suggestion and the example. :) We will take this into consideration and I will create an internal issue to track this.

**Reply by 2chiefk** [reply]
https://github.com/orgs/community/discussions/139726

**Reply by 2chiefk** [reply]
I know I love to be a wise ass to all I designed Android and Google stole it from me what did you all expect

**Reply by MFKDGAF** [reply]
I would like to add on to the original request. 
I noticed when sharing a user via email, that the subject line was not filled out in the email either. So I would like to suggest when sharing a user that the name of the user gets auto populated in the subject line if the email.

To summarize:

When sharing a repository via email, the subject line of the email should be pre populated with name of the repository and the body of the email should be pre populated with a link to the repository. 

When sharing a user via email, the name of the user should be pre populated in the subject line of the email and the body of the email should be pre populated with a link to the user.

**Comment by 2chiefk** [comment]
Yeah you are welcome

**Comment by 2chiefk** [comment]
https://github.com/orgs/community/discussions/139726

**Comment by josh2318998** [comment]
I believe it's a great idea to enhance the sharing experience on GitHub by automatically populating the email subject with the repository name or the author's name combined with the repository title. This improvement would streamline communication, similar to how the iOS Reddit app efficiently fills in post titles. Such an [app](https://getreminiproapp.com/)-like feature would make it easier for recipients to understand the context of the shared repository right away. Would you consider implementing this enhancement?

**Comment by ruqiabegumm** [comment]
When sharing a repository via email, I agree that the default action should indeed fill the subject with the repository name or the author + repository name. This would streamline the sharing process and make the emails more informative and professional.

For instance, just as on the iOS Reddit mobile app, where the subject automatically includes the name of the post, this enhancement would provide better clarity for the recipient. Additionally, it saves the sender from manually editing the subject every time they share a repository.

If you’re looking for some thoughtful reading material while discussing changes, check out [Sad Shayari in Hindi](https://sadshayri.net/) – an ideal collection for those emotional moments.

This enhancement can be especially useful when repositories are shared internally within a team, ensuring smoother communication. Similar improvements in other platforms have shown that users appreciate minor but impactful changes.

If you want to explore more profound content, have a look at [Emotional Sad Shayari](https://sadshayri.net/300-emotional-sad-shayari-in-hindi/) – a compilation that resonates with deep emotions.

**Comment by Chingsu6282** [comment]
I agree that enhancing the default email sharing action to automatically fill in the subject line with relevant information, such as the name of the repository or the author and repository name, would be a great improvement. This could enhance clarity for the recipient and make the email more professional by default.

Taking inspiration from how apps like the iOS Reddit mobile app handle post sharing, where the name of the post is added to the subject, could be a good model to follow. It would streamline the sharing process and improve user experience overall. For example, if you're wondering [how to get free mod apk in Vietnam](https://bandishare.pro/   (2 Naked URLs)), this could also be shared effectively through a well-structured subject line.

**Comment by lewis8329832** [comment]
Thank you for the suggestion! Automatically populating the email subject with the repository name or a combination of the author and repository name would be a smart improvement. It would [stick the app](https://stickwarlegacyapp.com/) out by providing recipients with immediate context, making shared repositories more recognizable. Similar to how the iOS Reddit app sticks the post title in the subject when sharing, this would make communication clearer and more efficient. Implementing such a feature on GitHub would streamline and enhance the sharing experience.

**Comment by ButtlerDrownPF** [comment]
Good suggestion! Some apps, like [Brawl Stars APK,](https://brawlstarsmodapk.com/) make sharing smoother with pre-filled subjects. This feature would be a great addition here too!

**Comment by ButtlerDrownPF** [comment]
Great idea! Adding the repository name to the email subject, like how apps such as [Stick War App](https://stickwarlegacyapks.com/) streamline sharing, would make things clearer and more efficient on GitHub.

**Comment by WillByce** [comment]
I think it’d be awesome if GitHub could improve the sharing experience by automatically filling in the email subject with the repository name or the author’s name along with the repo title. It’d really help streamline communication, kind of like how the iOS Reddit app smartly auto-fills post titles. [App ](https://thevnpro.com/
)feature like this would make it super easy for people to get the context of a shared repo right off the bat. Any chance you’d consider adding something like this?

**Comment by MFKDGAF** [comment]
Any update on this? It has been 7 months since I've created this request.

**Comment by MFKDGAF** [comment]
> Good feedback @MFKDGAF. Thank you for the suggestion and the example. :) We will take this into consideration and I will create an internal issue to track this. 

Asking for an update on this.


I submitted this on September 25, 2024 and was really hoping this would have been implemented by now seeing as this is a simple and necessary function of the app.
